docs/source/pagination/introduction.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Pagination
+---
+
+Apollo Pagination provides a convenient and easy way to interact with and watch paginated APIs. It provides a flexible and powerful way to interact with paginated data, and works with both cursor-based and offset-based pagination. Its key features include:
+
+- Watching paginated data
+- Forward, reverse, and bidirectional pagination support
+- Multi-query pagination support
+- Support for custom model types
+
+Apollo Pagination provides two classes to interact with paginated endpoints: `GraphQLQueryPager` and `AsyncGraphQLQueryPager`. They have very similar APIs, but the latter supports `async`/`await` for use in asynchronous contexts.
+
+## Using a `GraphQLQueryPager`
+
+The `GraphQLQueryPager` class is intended to be a simple, flexible, and powerful way to interact with paginated data. While it has a standard initializer, it is recommended to use the convenience initializers, which simplify the process of creating a `GraphQLQueryPager` instance. The convenience initializers are static methods on the `GraphQLQueryPager` class, prefixed with `make`.
+
+For example, in order to initialize a `GraphQLQueryPager` that paginates a single query in the forward direction with a cursor-based pagination, you can use the `makeQueryPager` method:
+
+```swift
+let initialQuery = MyQuery(first: 10, after: nil)
+let pager = GraphQLQueryPager.makeQueryPager(
+    client: client,
+    initialQuery: initialQuery,
+    pageResolver: { page, paginationDirection in
+        // As we only want to support forward pagination, we can return `nil` for reverse pagination
```

Comment:
For the convenience functions that only support single direction paging, can we just omit the pagination direction from the `pageResolver` closure signature? It should only be needed for bidirectional paging, right?

Reply:
Discussed this on our call just now – but this can be an improvement for 0.2.x

We could do it by declaring `ForwardPageInfo` and `ReversePageInfo` protocols, and then declaring the convenience functions around those protocols.